pages/gallery.jsx

```diff
@@ -6,6 +6,8 @@ import TextSection from "components/EvSections/TextSection";
 import EvLayout from "components/layouts/EvLayout";
 
 import api from "utils/api/grocery3-shop";
+import RoundedImagesPanel from "components/EvComponents/RoundedImages/RoundedImagesPanel";
```

Comment:
remove the RoundedImagesPanel from the EvComponents to EvSections and change its name to RoundedImagesSection

Reply:
changed position and name